examples/Demo/Shared/Pages/Dialog/Examples/DialogCustomHeaderFooter.razor

```diff
@@ -0,0 +1,72 @@
+﻿@inject IDialogService DialogService
+
+
+<div style="margin-top: 1rem;">
+    <FluentButton @onclick="@OpenDialogAsync" Appearance="Appearance.Accent">
+        Open Dialog
+    </FluentButton>
+</div>
+
+@code {
+    private bool _trapFocus = true;
+    private bool _modal = true;
+    private IDialogReference dialog;
+
+    SimplePerson simplePerson = new()
+        {
+            Firstname = "Dan",
+            Lastname = "Sanderson",
+            Age = 42,
+        };
+
+    private async Task OpenDialogAsync()
+    {
+        DemoLogger.WriteLine($"Open dialog left");
+
+        DialogParameters<SimplePerson> parameters = new()
+            {
+                HeaderTemplate = new RenderFragment(builder =>
```

Comment:
It's not easy to create an HTML part using C#. Why not to include this part into the `SimpleFormInDialog.razor` file?
For example, why not to simplify this usage with Tag/CSS styles

**SimpleFormInDialog.razor**
```xml
@implements IDialogContentComponent<SimplePerson>

<FluentDialogHeader ShowDismiss="true" Title="Hello Man"> 
   ...
<FluentDialogHeader>

<FluentDialogFooter PrimaryAction="Save"> 
   ...
<FluentDialogFooter

<EditForm Model="@Content" OnInvalidSubmit="HandleInvalid" OnValidSubmit="HandleValid">
    <FluentMessageBarContainer Section="@App.MESSAGES_DIALOG" MaxMessageCount="1" />

    <FluentTextField @bind-Value="@Content.Firstname">Your firstname:</FluentTextField>
    <FluentTextField @bind-Value="@Content.Lastname">Your lastname:</FluentTextField>
    <FluentNumberField @bind-Value="@Content.Age" Min="10" Max="110" >Your age:</FluentNumberField>
</EditForm>
```